.github/workflows/cd.yml

```diff
@@ -24,3 +24,15 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - name: get-npm-version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@master
+      - uses: google-github-actions/setup-gcloud@master
+        with:
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: Sync dist
```

Comment:
This is copied from the excel add in and I've tested it on this branch and the files are in this bucket. But I copied the settings that I could see from the excel add-in bucket and for some reason the files in that one open via their "public url" fine, even in an incognito window, but the files in this new bucket give me "access denied" and I'm not sure what I'm missing.

Reply:
I believe it was Dag who created the `crisp-web-assets` bucket for us and assigned the "public object" permissions to the bucket. I found [this](https://stackoverflow.com/a/45869920/16109045)  on SO but we don't have permissions to access bucket permissions 🙂 You will probably need to reach out to Justin, Dag or Li to make these files publicly available.